ttkwidgets/itemscanvas.py

```diff
@@ -103,7 +105,25 @@ def left_motion(self, event):
         rectangle = self.items[item]
         self.config(cursor="exchange")
         self.canvas.itemconfigure(item, fill="blue")
-        x, y = self.canvas.canvasx(event.x), self.canvas.canvasy(event.y)
+        xc, yc = self.canvas.canvasx(event.x), self.canvas.canvasy(event.y)
+        dx, dy = xc - self.current_coords[0], yc - self.current_coords[1]
+        self.current_coords = xc, yc
+#        x = self._max_x if x > self._max_x else x
```

Comment:
This section of comments contains the code in lines 127-130. I'm not sure why it's here though?

Reply:
I had just commented out the old code and forgot to remove it. It's done now.